pomegranate/distributions.pyx

```diff
@@ -49,7 +49,7 @@ def log(value):
 	if isinstance( value, numpy.ndarray ):
 		to_return = numpy.zeros(( value.shape ))
 		to_return[ value > 0 ] = numpy.log( value[ value > 0 ] )
-		to_return[ value == 0 ] = NEGINF
+		to_return[ numpy.isclose(value, 0) ] = NEGINF
```

Comment:
this is covered by `replace floating point == 0 tests with isclose while retain all tests passing` goal, more justification may be applied.